_posts/en/newsletters/2024-01-31-newsletter.md

```diff
@@ -0,0 +1,253 @@
+  when they cannot both exist in the same valid block chain, usually
+  because they both try to spend the same input---violating the rule
+  that an input can only be spent once within a particular block chain.
+  The rules for [RBF][topic rbf] compare a transaction in the
+  mempool against a newly received transaction that conflicts with it.
+  Zhao suggests an idealized way to think about a conflicts policy: if
+  you have two transactions but you can only accept one, you should not
+  choose the one that arrives first---you should choose the one that
+  best suits your goals (such as maximizing miner revenue without
+  allowing effectively free relay).  The RBF rules attempt to do that
+  for conflicts, Zhao in her post extends the idea to related
+  transactions rather than just conflicts.
+
+  Bitcoin Core places _policy_ limits on the number and size of related
+  transactions that are concurrently allowed in the mempool.  This
+  mitigates several DoS attacks, but means that it might reject
+  transaction B because it previously received related transaction A,
+  which maxed out the limits.  This violates Zhao's principle: instead,
+  Bitcoin Core should accept whichever of A or B is actually best for
+  its goals.
+
+  The proposed rules for [v3 transaction relay][topic v3 transaction
+  relay] only allow an unconfirmed v3 parent to have a single child
+  transaction in the mempool.  Applying the existing RBF rules to
+  replacements of that child is easy and Zhao has [implemented it][zhao
+  kindredimpl].  If, as described in [last week's newsletter][news286
+  imbued], existing LN commitment transactions using [anchors
+  outputs][topic anchor outputs] are automatically enrolled in the v3
+  policy, this ensures either party can always fee bump the commitment
+  transaction:
+
+  - Alice can send the commitment transaction with a child transaction
+    to pay fees.
+
+  - Alice can later RBF her existing child transaction to increase the
+    fees.
+
+  - Bob can use kindred replacement to evict Alice's child by sending a
+    child of his own that pays higher fees.
+
+  - Alice can later use kindred replacement on Bob's child by sending a
+    child of hers with an even higher fee (removing Bob's child).
+
+  Adding this policy and automatically applying it to current LN anchors
+  will allow the [CPFP carve-out rule][topic cpfp carve out] to be
+  removed, which is necessary for [cluster mempool][topic cluster
+  mempool] to be implemented, which should allow making replacements
+  of all kinds more incentive compatible in the future.
+
+  As of this writing, there were no objections to the idea on the forum.
+  One notable question was about whether this eliminated the need for
+  [ephemeral anchors][topic ephemeral anchors], but the author of that
+  proposal (Gregory Sanders) replied, "I have no plans on dropping
+  ephemeral anchor work.  Zero-satoshi outputs have a number of
```

Comment:
```suggestion
  ephemeral anchor work.  Zero-satoshi outputs have several
```

Reply:
Not changing a direct quote for this.

Reply:
Oh sorry, my mistake, I didn't notice this was within a quote, I agree.